pkg/controller/v1alpha2/applicationconfiguration/component.go

```diff
@@ -144,7 +144,8 @@ func (c *ComponentHandler) createControllerRevision(mt metav1.Object, obj runtim
 	// set annotation to component
 	revision := appsv1.ControllerRevision{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: revisionName,
+			Name:      revisionName,
+			Namespace: curComp.Namespace,
```

Comment:
could you help split the namespace in a separated PR as a BUG fix, thanks.

Reply:
Sure, I will fix it and modify unit tests in another PR.